docs/Home/index.js

```diff
@@ -38,29 +38,37 @@ const Foo = () => (
       `}
     </Code>
     <h2>Contribute</h2>
-    <p>
-      Check out our{' '}
-      <Link href="https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md">
-        contribution guide!
-        <Icon type="external-link" />
-      </Link>
-    </p>
-    <p>
-      Found a bug?{' '}
-      <Link href="https://github.com/dequelabs/cauldron/issues/new/choose">
-        Open an issue!
-        <Icon type="external-link" />
-      </Link>
-    </p>
+    <ul>
+      <li className="is--flex-row">
+        <p>Check out our</p>
+        <Link
+          className="Home__links"
+          href="https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md"
+        >
+          contribution guide!
+          <Icon type="external-link" />
```

Comment:
I think we also want to add a label so that screen reader users who can't see the icon know it's an external link.

`<Icon type="external-link" label="link leads to external site" />`

or 

`<Icon type="external-link" label="external site" />`

`

> It can be helpful to users to know when a link will take them away from the current web site to a different web site. Including a text indication (e.g. "link leads to an external site") or an icon with equivalent alt text are two of the most common ways to indicate this. Ensure the indication is available to both sighted users and blind screen reader users.

Reply:
Used `aria-label="{link text} - opens an external site"`